test/core/Gatekeeping.t.sol

```diff
@@ -739,15 +738,15 @@ contract GatekeepingTest is MangroveTest {
   function olReadOK(OLKey memory olKey) external {
     assertTrue(!mgv.locked(olKey), "market must not be locked");
     mgv.config(olKey);
+    mgv.global();
```

Comment:
or if it's intentional I don't get it 😅

Reply:
It's intentional: I want to ensure that the read-lock doesn't affect `global()` :-)
Maybe deserves a comment?

The functions are called in the order they are listed in `MgvView` for easy comparison and maintenance.

Reply:
Addressed in ffa58ba13c4d283e5f57df184a6dcc625b8b66e6